game.py

```diff
@@ -811,6 +965,12 @@ def get_move_consts():
                              "map_height": PacBot.MAP_HEIGHT,
                              "map_width": PacBot.MAP_WIDTH,
                              })
+    
+    @staticmethod
+    def map_distance(pos, target):
```

Comment:
This doesn't take into consideration map wrapping. Should it?

Reply:
The reference wasn't clear on that, I interpreted this image to suggest that it probably doesn't account for wrapping:

https://media.gameinternals.com/pacman-ghosts/decision.png